flutter/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [0.0.1] - TODO: Add release date.
```

Comment:
We should probably go ahead and fill this out, at least as far as we can

Reply:
I think we could remove this file and use tag description to document any changes if needed. https://github.com/microsoft/fluentui-system-icons/releases/tag/1.1.42

Reply:
@willhou : Addressed :)

Reply:
@willhou : This is also an important file , just added it back, let me know if the content is correct.  https://dart.dev/tools/pub/publishing#important-files